src/sidebar/util/fetch-config.js

```diff
@@ -132,9 +132,44 @@ async function fetchConfigRpc(appConfig, parentFrame, origin) {
     [],
     3000
   );
-  return fetchConfigEmbed(appConfig, remoteConfig);
+  // Closure for the RPC call for mutateAsyncGroups to scope parentFrame
```

Comment:
```suggestion
  // Closure for the RPC call for mutateAsyncGroups to scope parentFrame
```

The name `mutateAsyncGroups` is out of date. Maybe just remove this comment since I think it is easy enough to understand what it does by reading the code.